shared/src/simulation/behaviors/manage-patient-transport-to-hospital.ts

```diff
@@ -0,0 +1,510 @@
+}
+
+class VehiclesForPatients {
+    @IsValue('vehiclesForPatients')
+    readonly type = 'vehiclesForPatients';
+
+    @IsString({ each: true })
+    readonly red: readonly string[] = ['RTW'];
+
+    @IsInt()
+    @Min(0)
+    readonly redIndex: number = 0;
+
+    @IsString({ each: true })
+    readonly yellow: readonly string[] = ['RTW'];
+
+    @IsInt()
+    @Min(0)
+    readonly yellowIndex: number = 0;
+
+    @IsString({ each: true })
+    readonly green: readonly string[] = ['RTW', 'KTW'];
+
+    @IsInt()
+    @Min(0)
+    readonly greenIndex: number = 0;
+
+    static readonly create = getCreate(this);
+}
+
+export class ManagePatientTransportToHospitalBehaviorState
+    implements SimulationBehaviorState
+{
+    @IsValue('managePatientTransportToHospitalBehavior')
+    readonly type = 'managePatientTransportToHospitalBehavior';
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly id: UUID = uuid();
+
+    @IsOptional()
+    @IsUUID(4, uuidValidationOptions)
+    public readonly requestTargetId?: UUID;
+
+    @IsUUIDSet()
+    public readonly simulatedRegionsToManage: UUIDSet = {};
+
+    @IsPatientsPerUUID()
+    public readonly patientsExpectedInRegions: PatientsPerRegion = {};
+
+    @Type(() => PatientsTransportPromise)
+    @ValidateNested()
+    public readonly patientsExpectedToStillBeTransportedByRegion: readonly PatientsTransportPromise[] =
+        [];
+
+    @Type(() => VehiclesForPatients)
+    @ValidateNested()
+    public readonly vehiclesForPatients: VehiclesForPatients =
+        VehiclesForPatients.create();
+
+    /**
+     * @deprecated Use {@link updateRequestVehiclesDelay} instead
+     */
+    @IsInt()
+    @Min(0)
+    public readonly requestVehiclesDelay: number = 60 * 1000;
+
+    /**
+     * @deprecated Use {@link updateRequestPatientCountsDelay} instead
+     */
+    @IsInt()
+    @Min(0)
+    public readonly requestPatientCountsDelay: number = 15 * 60 * 1000;
+
+    @IsInt()
+    @Min(0)
+    public readonly promiseInvalidationInterval: number = 30 * 60 * 1000;
+
+    @IsLiteralUnion(patientStatusAllowedValues)
+    public readonly maximumCategoryToTransport: PatientStatusForTransport =
+        'red';
+
+    @IsBoolean()
+    public readonly transportStarted: boolean = false;
+
+    @IsOptional()
+    @IsUUID(4, uuidValidationOptions)
+    readonly recurringPatientDataRequestActivityId?: UUID;
+
+    @IsOptional()
+    @IsUUID(4, uuidValidationOptions)
+    readonly recurringSendToHospitalActivityId?: UUID;
+
+    static readonly create = getCreate(this);
+}
+
+export const managePatientTransportToHospitalBehavior: SimulationBehavior<ManagePatientTransportToHospitalBehaviorState> =
+    {
+        behaviorState: ManagePatientTransportToHospitalBehaviorState,
+        handleEvent: (draftState, simulatedRegion, behaviorState, event) => {
+            switch (event.type) {
+                case 'tickEvent': {
+                    // initialize recurring activities if needed
```

Comment:
Since you've now got a boolean indicating whether the transport is started, registering this could take place only if the boolean is set to `true`. This way, we'll ensure that the first occurence of the recurring events will be just at the time the transport has been started and not up to *interval length* seconds later.

The activities should also be terminated when the transport is stopped or the behavior gets removed from the simulated region.